ceph/mds.go

```diff
@@ -183,3 +217,204 @@ func (m *MDSCollector) Collect(ch chan<- prometheus.Metric, version *Version) {
 		}
 	}
 }
+
+type healthDetailCheck struct {
```

Comment:
this needs to parse the `mutes` dict as well:
```
    {                                                                                                                                                         
      "code": "OSDMAP_FLAGS",
      "ttl": "2024-02-15T18:14:00.953489+0000",                                
      "sticky": false,                                                                                                                                        
      "summary": "noout,norebalance,noscrub,nodeep-scrub flag(s) set",         
      "count": 38    
    },                          
    {             
      "code": "POOL_TOO_FEW_PGS",                                              
      "sticky": true,
      "summary": "",
      "count": 0                                                               
    },               
    {               
      "code": "POOL_TOO_MANY_PGS",
      "sticky": true,           
      "summary": "",
      "count": 0                                                               
    }
```

Reply:
Ah I forgot to mention, the `muted` field is boolean and is currently captured, so we can do this without needing to change the structure much.